tests/drivers/watchdog/wdt_error_cases/src/main.c

```diff
@@ -51,6 +51,15 @@
 #define MAX_INSTALLABLE_TIMEOUTS (8)
 #define WDT_WINDOW_MAX_ALLOWED   (0x07CFFFFFU)
 #define DEFAULT_OPTIONS          (WDT_OPT_PAUSE_IN_SLEEP | WDT_OPT_PAUSE_HALTED_BY_DBG)
+#elif defined(CONFIG_SOC_NRF54H20)
```

Comment:
Why not use `#if defined(CONFIG_SOC_NRF54L15) || defined(CONFIG_SOC_NRF54H20)`? The configuration seems to be the same.

Reply:
I thought that nrf54l15 supports WDT_FLAG_RESET_SOC
while nrf54h20 supports WDT_FLAG_RESET_CPU_CORE.
Therefore I started with separate configurations.
Test results show that I was wrong.

I will apply change as suggested.